src/citrine/resources/file_link.py

```diff
@@ -229,7 +229,7 @@ def _as_dict_from_resource(self, file: dict):
         }
         return file_dict
 
-    def upload(self, file_path: str, dest_name: str = None) -> FileLink:
+    def upload(self, file_path: str, *, dest_name: str = None) -> FileLink:
```

Comment:
It's true that the user can get away without specifying dest_name, but they'll often want to specify it. I think that both arguments are important enough that it doesn't make sense to elevate `file_path`

Reply:
changing it for the other methods in this file that weren't consistent too